superset/assets/src/visualizations/deckgl/layers/Arc/Arc.jsx

```diff
@@ -29,6 +31,18 @@ function getPoints(data) {
   return points;
 }
 
+function setTooltipContent(formData) {
+  return o => (
+    <div className="deckgl-tooltip">
+      <div>{`${t('Start (Longitude, Latitude)')}: `}<strong>{`${o.object.sourcePosition[0]}, ${o.object.sourcePosition[1]}`}</strong></div>
```

Comment:
It could be good to refactor into a `<TooltipKeyValue key={} label={} value={} />` or "`TooltipRow`" component, maybe the props are like key and value. Eventually it can be made better with a grid (right-align key, left align value), number formatter(s), ...

Reply:
Done!